From d0de3decf25245cbb1ae11b6eba1845079c86688 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 4 Oct 2016 20:14:38 -0400
Subject: [PATCH 1309/1722] drm/amd/dal: Disable HBR3 and TPS4

This is not working as well as we'd like. Disabling
for now.

Change-Id: I98ced97b47d82d0251ff3ff93419119721e8fdd0
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce112/dce112_link_encoder.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce112/dce112_link_encoder.c b/drivers/gpu/drm/amd/dal/dc/dce112/dce112_link_encoder.c
index c048d35..97350f5 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce112/dce112_link_encoder.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce112/dce112_link_encoder.c
@@ -179,9 +179,9 @@ bool dce112_link_encoder_construct(
 
 	enc110->base.funcs = &dce112_lnk_enc_funcs;
 
-	enc110->base.features.flags.bits.IS_HBR3_CAPABLE = true;
+	enc110->base.features.flags.bits.IS_HBR3_CAPABLE = false;
 
-	enc110->base.features.flags.bits.IS_TPS4_CAPABLE = true;
+	enc110->base.features.flags.bits.IS_TPS4_CAPABLE = false;
 
 	return true;
 }
-- 
2.7.4

